week-1-ts-ecosystem/w01e01-typescript-basics/src/main.ts

```diff
@@ -1,28 +1,5 @@
-/**
- * Zasady co do pliku:
- *
- * Możesz dowolnie modyfikować zawartość tego pliku,
- * całość programu musi jednak działać tak jak do tej pory !
- *
- * */
-
-const user: any = {
-  name: 'Andy',
-  age: 30,
-  email: 'andy@mail-me-tommorow.com',
-  address: {
-    street: 'Strange Alley',
-    no: 23,
-  },
-}
-
-function hasAddress(user: any): any {
-  return Boolean(user.address)
-}
-
-function hasGivenAge(requiredAge: any): any {
-  return (user: any): any => user.age >= requiredAge
-}
+import {user} from './data'
+import {hasAddress, hasGivenAge} from './logic'
 
 const isAdult = hasGivenAge(18)
```

Comment:
TO powinno iśc do walidatorów, a dodatkowo, to tutaj mogło zostać całe `const user: User = ` tutaj może zostać "data" można dociągnąć validators i je użyć, tak jak było:
```javascript
console.log(`User ${user.name} is ${isAdult(user) ? 'adult' : 'minor'}`)
console.log(`and has${hasAddress(user) ? '' : ' no'} address`)
```

Bo gdzieś Ci to "uciekło" z kodu (nie widzę tego w kodzie u Ciebie)